src/vm/amd64/UMThunkStub.asm

```diff
@@ -279,6 +283,21 @@ DoThreadSetup:
         
         jmp             HaveThread
         
+InvalidTransition:
+        mov             [rbp + UMThunkStubAMD64_ARGUMENTS_STACK_HOME_OFFSET +  0h], rcx
+        mov             [rbp + UMThunkStubAMD64_ARGUMENTS_STACK_HOME_OFFSET +  8h], rdx
```

Comment:
Could you please remove this setup as well - to match what you have done for Unix?